replica/database.hh

```diff
@@ -1489,7 +1489,12 @@ private:
     size_t max_memory_concurrent_view_update_reads() { return _dbcfg.available_memory * 0.01; }
     // Assume a queued read takes up 1kB of memory, and allow 2% of memory to be filled up with such reads.
     size_t max_inactive_queue_length() { return _dbcfg.available_memory * 0.02 / 1000; }
-    size_t max_inactive_view_update_queue_length() { return _dbcfg.available_memory * 0.01 / 1000; }
+    size_t max_inactive_view_update_queue_length() {
+        if (const auto max = utils::get_local_injector().inject_parameter<size_t>("max_view_update_read_queue"); max) {
```

Comment:
Nitpick: why does the function and the injection forcing it to return a different value have such different names? Perhaps one of them is mis-named?